src/Network/VCR/Types.hs

```diff
@@ -140,5 +148,28 @@ toHeader (name, value)  =  (mk $ BE.encodeUtf8 name, BE.encodeUtf8 value)
 
 
 emptyCassette :: Text -> Cassette
-emptyCassette endpoint = Cassette { endpoint = endpoint, apiCalls = [], ignoredHeaders = [] }
-
+emptyCassette endpoint = Cassette { endpoint = endpoint, apiCalls = [], ignoredHeaders = [], ignoredBodyFields = [] }
+
+-- utility methods for matching with ignored fields in the body
+
+-- | Remove the ignored fields from the request's body
+modifyBody :: [Text] -> SavedRequest -> SavedRequest
+modifyBody ignoredBodyFields (SavedRequest mn hs url ps body) = SavedRequest mn hs url ps body'
+  where ignoredBodyFieldsMap = HashMap.fromList $ zip ignoredBodyFields (repeat Null)
+        bodyMap :: HashMap Text Value = HashMap.difference (fromJust . decode $ L.fromStrict body) ignoredBodyFieldsMap
```

Comment:
What do we want to do if the body doesn't have the expected shape? Should probably be a better error than `fromJust` :)